blog.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <title>Adopt-a-dog | Dog Adoption</title>
+    <link rel="stylesheet" href="styles/style.css">
+  </head>
+  <body>
+    <div class="header-img">
+      <a href="index.html">
+      <img class="logo" src="images/adoptadog-logo.svg" alt="Adopt-a-dog logo.">
```

Comment:
This will need to be indented in one level from the anchor tag

Reply:
Done